components/brave_rewards/browser/rewards_service_observer.h

```diff
@@ -77,10 +77,8 @@ class RewardsServiceObserver : public base::CheckedObserver {
   virtual void OnPublisherListNormalized(
       RewardsService* rewards_service,
       brave_rewards::ContentSiteList list) {}
-  virtual void OnAdsNotificationsData(
-      brave_rewards::RewardsService* rewards_service,
-      int total_viewed,
-      double estimated_earnings) {}
+  virtual void OnConfirmationsHistoryChanged(
```

Comment:
why does this have a different name `ConfirmationsHistory` vs `AdsNotificationHistory`? That seems confusing to me and I think we should use consistent naming. Also I think we should still pass the total_viewed and estimated_earnings through here

Reply:
done